scripts/install

```diff
@@ -42,85 +41,80 @@ fi
 final_path="/var/www/$app"
 test ! -e "$final_path" || ynh_die "This path already contains a folder"
 
-# Normalize the url path syntax
-path_url=$(ynh_normalize_url_path "$path_url")
-
 # Check web path availability
 ynh_webpath_available "$domain" "$path_url"
+
 # Register (book) web path
-ynh_webpath_register "$app" "$domain" "$path_url"
+ynh_webpath_register --app=$app --domain=$domain --path_url=$path_url
 
 #=================================================
 # STORE SETTINGS FROM MANIFEST
 #=================================================
+ynh_script_progression --message="Storing installation settings..."
 
-ynh_app_setting_set "$app" domain "$domain"
-ynh_app_setting_set "$app" path "$path_url"
+ynh_app_setting_set --app=$app --key=domain --value=$domain
+ynh_app_setting_set --app=$app --key=path --value=$path_url
 
 #=================================================
 # DEFAULT SETTINGS
 #=================================================
+ynh_script_progression --message="Configuring firewall..."
 
-port=$(ynh_find_port 9876)
+port=$(ynh_find_port --port=9876)
 salt=$(ynh_string_random 40)
 
-ynh_app_setting_set "$app" port "$port"
-ynh_app_setting_set "$app" salt "$salt"
+ynh_app_setting_set --app=$app --key=port --value=$port
+ynh_app_setting_set --app=$app --key=salt --value=$salt
 
 #=================================================
 # STANDARD MODIFICATIONS
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-
 ynh_script_progression --message="Installing app dependencies"
-ynh_install_app_dependencies $pkg_dependencies
+
+ynh_exec_warn_less ynh_install_app_dependencies $pkg_dependencies
 
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
+ynh_script_progression --message="Setting up source files..."
 
-ynh_app_setting_set "$app" final_path "$final_path"
-
+ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 # Download, check integrity, uncompress and patch the source from app.src
-ynh_script_progression --message="Downloading app from sources"
-ynh_setup_source "$final_path"
+ynh_setup_source --dest_dir="$final_path"
+
 mkdir "$final_path/data"
 
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
+ynh_script_progression --message="Configuring NGINX web server..."
 
 # Create a dedicated nginx config
 ynh_add_nginx_config
 
-if [ "$path_url" != "/" ]
```

Comment:
One last question: can you explain why this is not needed anymore please?

Reply:
That's included in NGINX helper.
https://github.com/YunoHost/yunohost/blob/1279755463100e6c4a0ba7d4b50f8f39658c8eac/data/helpers.d/nginx#L60